Backend/src/controllers/solana.controller.ts

```diff
@@ -47,53 +45,11 @@ export const createDepositTransaction = async (req: Request, res: Response) => {
         })),
         data: depositIx.data.toString("base64"),
       },
-    ];
-
-    return res.status(200).json({
-      instructions: serializedInstructions,
-      message: "Deposit instruction ready — user must sign",
     });
   } catch (err) {
     console.error("Error creating deposit transaction:", err);
-    return res.status(500).json({
-      error: "Failed to create deposit transaction",
-      details: err instanceof Error ? err.message : String(err),
-    });
-  }
-};
-
-export const initializeConfig = async (req: Request, res: Response) => {
-  try {
-    const { adminPubkey } = req.body;
-    if (!adminPubkey) {
-      return res.status(400).json({ error: "Missing adminPubkey" });
-    }
-
-    const adminKey = new PublicKey(adminPubkey);
-    // For testing, adminPubkey is the wallet address (you sign from frontend)
-    const initIx = await createInitializeConfigInstruction(
-      adminKey,
-      1000,
-      1,
-      adminKey,
-    );
-
-    // Serialize instruction to send to frontend
-    const serializedInstruction = {
-      programId: initIx.programId.toBase58(),
-      keys: initIx.keys.map((k) => ({
-        pubkey: k.pubkey.toBase58(),
-        isSigner: k.isSigner,
-        isWritable: k.isWritable,
-      })),
-      data: initIx.data.toString("base64"),
-    };
-
-    res.status(200).json({ instructions: [serializedInstruction] });
-  } catch (err) {
-    console.error("Error creating initializeConfig instruction:", err);
-    res
+    return res
       .status(500)
-      .json({ error: "Failed to create initializeConfig instruction" });
+      .json({ error: "Failed to create deposit transaction" });
   }
 };
```

Comment:
hi @Devanshgoel-123, this initialization function is needed for the program.

Reply:
my bad! 

i checked again now. the correct function in use is `enusreConfigInitialized`. we can disregard this.